apps/web/components/pages/task/description-block/editor-components/BlockButton.tsx

```diff
@@ -15,27 +17,23 @@ const BlockButton = ({
 	format,
 	icon: Icon,
 	isBlockActive,
+	className,
 }: IMarkButtonProps) => {
 	const editor = useSlate();
 
 	return (
 		<button
-			className="my-1"
-			style={{
-				background: isBlockActive(
```

Comment:
@desperado1802 Use `useMemo` hook to store `isBlockActive` function's value and then use that variable to dynamically render classes